.rubocop_todo.yml

```diff
@@ -192,6 +192,14 @@ Style/ExpandPathArguments:
     - 'lib/generators/arclight/install_generator.rb'
     - 'lib/generators/arclight/update_generator.rb'
 
+# Offense count: 1
+# Cop supports --auto-correct.
+# Configuration parameters: EnforcedStyle.
+# SupportedStyles: always, never
+Style/FrozenStringLiteralComment:
```

Comment:
Depending on the resolution of my other comment, should we just go ahead and add the magic comment (or disable it in the regular rubocop file)?

Reply:
👍 gotcha.. yeah updated and force pushed.